vpr/src/util/vpr_utils.h

```diff
@@ -30,9 +30,14 @@ bool is_empty_type(t_logical_block_type_ptr type);
 //Returns the corresponding physical type given the logical type as parameter
 t_physical_tile_type_ptr physical_tile_type(ClusterBlockId blk);
 
+//Returns the sub tile corresponding to the logical block location within a physical type
+int get_sub_tile_index(ClusterBlockId blk);
+
 int get_unique_pb_graph_node_id(const t_pb_graph_node* pb_graph_node);
 
-void get_class_range_for_block(const ClusterBlockId blk_id, int* class_low, int* class_high);
+void get_class_range_for_block(const ClusterBlockId blk_id,
+                               int* class_low,
+                               int* class_high);
```

Comment:
I saw there was a t_class_range. Should that be used here?

Reply:
Done